Sources/Plasma/NucleusLib/pnNetBase/pnNbConst.h

```diff
@@ -49,15 +49,11 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 *
 ***/
 
-// FIXME: Remove all Windows assumptions in the net code
-#ifndef MAX_PATH
-#   define MAX_PATH 260
-#endif
-
```

Comment:
Tossed this. This define conflicted with the other one, and we don't use MAX_PATH in the netcode anymore.